lib/matplotlib/__init__.py

```diff
@@ -1491,6 +1491,17 @@ def verify_test_dependencies():
     if not os.path.isdir(os.path.join(os.path.dirname(__file__), 'tests')):
         raise ImportError("matplotlib test data is not installed")
 
+    # The version of freetype to install locally for running the
+    # tests.  This must match the value in `setupext.py`
+    LOCAL_FREETYPE_VERSION = '2.5.2'
+
+    from matplotlib import ft2font
+    if (ft2font.__freetype_version__ != LOCAL_FREETYPE_VERSION or
+        ft2font.__freetype_build_type__ != 'local'):
+        raise ImportError(
```

Comment:
Is it feasible to allow running the tests, just giving a big warning and marking all image comparisons as known-fail? That would still let people run the other tests, or look at the results of single image-comparison tests manually, while not leading them to think the test results are canonical.